plugins/services/src/js/structs/Service.d.ts

```diff
@@ -1,11 +1,14 @@
 import Item from "#SRC/js/structs/Item";
 import { Status } from "#PLUGINS/services/src/js/constants/ServiceStatus";
+import { ServiceQuotaRolesStats } from "../types/ServiceQuotaRolesStats";
 
 export default class Service extends Item {
   getId(): string;
   getMesosId(): string;
   getName(): string;
   getRole(): string;
+  getRootGroupName(): string;
```

Comment:
this is true, because every serviceTree or service will always contain a `/`, right?